extensions/index.js

```diff
@@ -4,16 +4,12 @@ const path = require('path');
 const extensions = {};
 
 fs.readdirSync(__dirname).forEach(moduleDir => {
-    const extStat = fs.statSync(path.join(__dirname, moduleDir));
-    if (extStat.isDirectory()) {
-        let indexStat;
-        try {
-            indexStat = fs.statSync(
-                path.join(__dirname, moduleDir, 'index.js'));
-        } catch (err) {} // eslint-disable-line no-empty
-        if (indexStat) {
-            extensions[moduleDir] = require(`./${moduleDir}`);
-        }
+    const extPath = path.join(__dirname, moduleDir);
+    const indexPath = path.join(extPath, 'index.js');
+    
+    if (fs.statSync(extPath).isDirectory() && 
```

Comment:
this may still throw, if the file (or directory!) does not exist: either pass `throwIfNoEntry` option, or wrap with try/catch

Reply:
Calling `existsSync` directly works fine